Mage/src/main/java/mage/abilities/AbilityImpl.java

```diff
@@ -558,7 +558,11 @@ protected VariableManaCost handleManaXCosts(Game game, boolean noMana, Player co
                     } else {
                         String manaSymbol = null;
                         if (variableManaCost.getFilter().isBlack()) {
-                            manaSymbol = "B";
+                            if (variableManaCost.getFilter().isRed()) {
+                                manaSymbol = "B/R";
```

Comment:
Are these two lines above required for the card to work? Seems pretty hacky and makes me think there should be a better way to accomplish this if so - otherwise we have an absurd amount of additional `if` statements to cover all color combinations.